.rubocop_todo.yml

```diff
@@ -13,40 +13,33 @@ Lint/AmbiguousRegexpLiteral:
     - 'test/vagrant-service-manager/binary_handlers/adb_openshift_binary_handler_test.rb'
 
 # Offense count: 1
-Lint/HandleExceptions:
+Lint/Eval:
   Exclude:
-    - 'lib/vagrant-service-manager/service.rb'
+    - 'features/support/env.rb'
 
-# Offense count: 13
+# Offense count: 17
 Metrics/AbcSize:
   Max: 53
 
-# Offense count: 1
-# Configuration parameters: CountComments.
-Metrics/ClassLength:
-  Max: 209
-
-# Offense count: 1
+# Offense count: 4
 Metrics/CyclomaticComplexity:
   Max: 28
 
-# Offense count: 13
+# Offense count: 17
 # Configuration parameters: CountComments.
 Metrics/MethodLength:
   Max: 96
 
 # Offense count: 1
 # Configuration parameters: CountComments.
 Metrics/ModuleLength:
-  Max: 111
-  Exclude:
-    - 'lib/vagrant-service-manager/plugin_util.rb'
+  Max: 132
```

Comment:
this will increase module length for all modules from now moving forward, opposed to exclude just _plugin_util.rb_, right?